src/celeritas/io/ImportProcess.hh

```diff
@@ -128,6 +128,7 @@ struct ImportProcess
     //!@}
 
     int                                      particle_pdg;
+    int                                      secondary_pdg{0};
```

Comment:
While you're at it, could you add default initializer to `particle_pdg` for safety?